include/picongpu/particles/ionization/byField/ADK/AlgorithmADK.hpp

```diff
@@ -132,11 +133,11 @@ namespace ionization
                 if( randNr < probADK )
                 {
                     /* return number of macro electrons to produce */
-                    return 1u;
+                    return IonizerReturn{iEnergy, 1u};
                 }
             }
             /* no ionization */
-            return 0u;
+            return IonizerReturn{0.0, 0u};
```

Comment:
This file seems to follow the other spacing style. For consistency please format as `IonizerReturn{ iEnergy, 1u };`